NorthstarDLL/mods/modmanager.h

```diff
@@ -9,6 +9,7 @@
 #include <filesystem>
 
 const std::string MOD_FOLDER_SUFFIX = "/mods";
+const std::string THUNDERSTORE_MOD_FOLDER_SUFFIX = "/thunderstore-mods";
```

Comment:
Actually maybe we should call it `mods-thunderstore` so it's closer to `mods` in the file manager when sorting alphabetically ^^

Reply:
I'd prefer something like `packages` to stay more in line with the current folder naming

Reply:
> I'd prefer something like `packages` to stay more in line with the current folder naming

I agree, it's probably best to keep thunderstore related things aligned with thunderstore's naming conventions and definitions. There is no such thing as a thunderstore mod in reality, the term came about simply to distinguish between mods that come from the discord server and mods that come from thunderstore.

Reply:
Should we do something like `thunderstore-packages` then or just `packages`? I have the feeling that `packages` on it's own might not be associated with modding for most users. The name _Thunderstore_ on the other side might be more associated with modding for someone who used Northstar previously. ^^

Reply:
Why does it need to be associated with modding? The user shouldn't be interacting with this folder anyway, except for manual installs who can figure it out by themselves and not be supported much further.

Reply:
fair point ^^

Reply:
Done in 83f93c25c09204d746a1c89693c6a94082fa761c

Reply:
We should've stuck with `thunderstore-packages`...

https://github.com/R2Northstar/Northstar/issues/537